### ISSUE TITLE
Landmarks are not provided in the page: A11y_.NetAspireDashboard_Resource_Resource_AI4W

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3361]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Tags:
#A11yTCS; #A11yMAS; #A11ySev3; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.3.1; #Element:Landmarks;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Hit the URL: http://localhost:15888/
2. TAB to "Resources" tab item and select the tab by pressing Enter key.
3. Turn on toggle button of landmarks in Accessibility Insights for Web extension.
4. Observe the issue.

### Actual Results:
Landmarks are not provided in the page.

**Similar issue is also observed in throughout the application.**

### Expected Results:
All landmarks should be provided on the page like main landmark, navigation landmark, etc.

### User Impact:
Screen reader users will not be able to understand the structure of the page.

### Attachment:
![3361_A11y_ NetAspireDashboard_Resource_Resource_AI4W](https://github.com/dotnet/aspire/assets/90901519/529c54e2-8b88-42b2-9770-72115d0682a5)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-09-24;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment

![image](https://github.com/user-attachments/assets/09e0bc57-dca6-4625-a030-5d1895d812c6)
